test/runtests.jl

```diff
@@ -22,3 +22,7 @@ include("nf.jl")
 include("interface.jl")
 include("fuzz.jl")
 include("adjoints.jl")
+
+if haskey(ENV, "CI")
+    include("benchmark.jl")
```

Comment:
Maybe use the buildkite benchmark nodes?

Reply:
Example config I can copy-paste?

Reply:
https://github.com/SciML/DiffEqBase.jl/blob/master/.buildkite/pipeline.yml change and change the queue part to match https://github.com/SciML/SciMLBenchmarks.jl/blob/f44a419b90d20b381c2cb89cafbb40671c39aa12/.buildkite/pipeline.yml#L19-L20